release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1445,17 +1443,17 @@ In the table, features are marked with the following statuses:
 |vSphere 6.7 Update 2 or earlier
 |GA
 |DEP
-|DEP
+|REM
 
 |Virtual hardware version 13
 |GA
 |DEP
-|DEP
+|REM
 
 |VMware ESXi 6.7 Update 3 or earlier
```

Comment:
VMware ESXi 7.0 update 1 - we should add that it's deprecated in this table for OpenShift 4.10.